app/views/support_interface/regions/edit.html.erb

```diff
@@ -5,29 +5,24 @@
 <%= form_with model: [:support_interface, @region] do |f| %>
   <%= f.govuk_error_summary %>
 
-  <%= f.govuk_check_box :application_form_skip_work_history, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Skip work history" } %>
-  <%= f.govuk_check_box :reduced_evidence_accepted, 1, 0, multiple: false, link_errors: true, small: true, label: { text: "Accept reduced evidence" } %>
-
-  <%= f.govuk_collection_select :sanction_check, [
-    OpenStruct.new(id: 'online', name: 'Online'),
-    OpenStruct.new(id: 'written', name: 'Written'),
-    OpenStruct.new(id: 'none', name: 'None')
-  ], :id, :name, label: { text: "Sanction check" } %>
+  <%= render "shared/application_form", f: %>
+  <%= render "shared/preliminary_check_form_fields", f: %>
+  <%= render "shared/eligibility_checker", f: %>
+  <%= render "shared/status_sanctions_checks", f: %>
+  <%= render "shared/teaching_authority_form_fields", f: %>
 
-  <%= f.govuk_collection_select :status_check, [
-    OpenStruct.new(id: 'online', name: 'Online'),
-    OpenStruct.new(id: 'written', name: 'Written'),
-    OpenStruct.new(id: 'none', name: 'None')
-  ], :id, :name, label: { text: "Status check" } %>
+  <p class="govuk-hint">Automatically notify the teaching authority (via email) every time a QTS application is submitted?</p>
+  <%= f.govuk_radio_button :eligibility_enabled, 'yes', label: { text: 'Yes' }, link_errors: true %>
+  <%= f.govuk_radio_button :eligibility_enabled, 'no', label: { text: 'No' } %>
```

Comment:
This would map to `teaching_authority_requires_submission_email` rather than `eligibility_enabled`.

Reply:
Could these radio buttons be put in govuk_collection_radio_buttons without a legend?

Reply:
I think the legend should be "Automatically notify the teaching authority (via email) every time a QTS application is submitted?". Having it as a paragraph with the hint class means that it's not linked to the radio buttons in any way, whereas having it as a legend in the fieldset it gets some additional functionality for accessibility. (Being able to tab between fields, or reading it out with screenreaders, etc).